docs/source/overview/delivery_services.rst

```diff
@@ -460,34 +460,6 @@ Free text field that has no strictly defined purpose, but it is suggested that i
 	| longDesc | Traffic Control source code and :ref:`to-api` responses | unchanged (``string``, ``String`` etc.) |
 	+----------+---------------------------------------------------------+-----------------------------------------+
 
-.. _ds-longdesc2:
```

Comment:
the fields should also be removed in the v4 API docs (`grep` for longDesc1 and longDesc2)